lib/job/arFindingAidJob.class.php

```diff
@@ -125,20 +152,116 @@ public function runJob($parameters)
 
     if ($exitCode != 0)
     {
-      $this->error('Converting the EAD FO to PDF has failed.');
-
+      $this->error($this->i18n->__('Converting the EAD FO to PDF has failed.'));
       $this->logCmdOutput($output, 'ERROR(FOP)');
 
       return false;
     }
 
-    $this->info("PDF generated successfully: $pdfPath");
+    $this->info($this->i18n->__('Finding aid generated successfully: %1', array('%1' => $pdfPath)));
 
     fclose($eadFileHandle); // Will delete the tmp file
     fclose($foFileHandle);
 
-    $this->job->setStatusCompleted();
-    $this->job->save();
+    return true;
+  }
+
+  private function upload($tempPath)
+  {
+    $this->info($this->i18n->__('Uploading finding aid (%1)...', array('%1' => $this->resource->slug)));
```

Comment:
Does it work when only using `%1`? Interesting, I always though it was needed to do `%1%` :)